src/flat-database.h

```diff
@@ -200,10 +200,10 @@ class CFlatDB
         int64_t nStart = GetTimeMillis();
 
         LogPrintf("Writing info to %s...\n", strFilename);
-        CoreWrite(objToSave);
+        const bool ret = CoreWrite(objToSave);
```

Comment:
We should probably not print "dump finished" if `ret` is false

Reply:
that log is for benchmark purposes, but I am ready to apply suggestions